Examples/RegistrationITKv4/DeformableRegistration17.cxx

```diff
@@ -102,8 +102,8 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "itkDisplacementFieldTransform.h"
 
 
-unsigned int RmsCounter = 0;
-double       MaxRmsE[4] = { 0.8, 0.75, 0.4, 0.2 };
+static unsigned int RmsCounter = 0;
```

Comment:
Maybe use the newer `inline` instead of `static`? See this [StackOverflow](https://stackoverflow.com/a/52887822/276168) answer.

Reply:
I'm no C++ expert, but aren't inline variables about having external linkage? Here this global variable is not needed outside this translation unit, so external linkage is not desired.

Reply:
Thanks @andrei-sandor, but I think it's still preferred in C++ to place those file-scope variables in an unnamed namespace, instead of declaring them `static`:
```cpp
namespace
{
unsigned int RmsCounter = 0;
double       MaxRmsE[4] = { 0.8, 0.75, 0.4, 0.2 };
} // namespace

```

It is no longer officially deprecated to use `static` here (https://stackoverflow.com/questions/4726570/deprecation-of-the-static-keyword-no-more), but it might still cause warnings from static analysis tools, if you do so.

Reply:
I don't really see how one is better/worse than the other, checking with `nm` the result is the same.  But if this is ITK's required style, we can change to that...

Reply:
I generally use the unnamed name space as it easily gets the correct linkage for variable, functions, and classes definitions.

Reply:
Thanks for the clang ticket @seanm! It describes the issue perfectly 💯